packages/ui/VirtualList/VirtualList.module.less

```diff
@@ -12,18 +12,18 @@
 		overflow-x: scroll;
 	}
 
-	.listItem {
-		overflow: hidden;
-		width: 100%;
-
-		will-change: transform;
-	}
-	
 	.content {
 		will-change: transform;
 	}
 }
 
+.listItem {
+	overflow: hidden;
+	width: 100%;
+
+	will-change: transform;
+}
```

Comment:
Why don't we add the `position: absolute` in the `listItem` class instead of the following inline style.

```
		composeStyle (width, height, primaryPosition, secondaryPosition) {
			const
				{x, y} = this.getXY(primaryPosition, secondaryPosition),
				style = {
					position: 'absolute',
```

Reply:
Good point!